data/json/recipes/recipe_armor.json

```diff
@@ -9398,7 +9398,33 @@
       [ "felt_patch", 8 ]
     ]
   ]
- },
+ },{
+  "type" : "recipe",
+  "result": "makeshift_knapsack",
+  "category": "CC_ARMOR",
+  "subcategory": "CSC_ARMOR_STORAGE",
+  "skill_used": "survival",
+  "difficulty": 0,
```

Comment:
Survival 1 recipes generally involve some investment and some real work rather than just an idea.
If the pants-knapsack would have survival 1, its primary role could easily be grinding survival (due to reversible recipe and negligible time).